gluten-ut/spark32/src/test/scala/io/glutenproject/utils/velox/VeloxTestSettings.scala

```diff
@@ -182,6 +182,8 @@ class VeloxTestSettings extends BackendTestSettings {
     .exclude("Map Concat")
     // Random.
     .exclude("Shuffle")
+    // TODO: ArrayDistinct should handle duplicated Double.NaN
+    .excludeByPrefix("SPARK-36741")
```

Comment:
@rui-mo Please note that velox array_distinct function is unable to handle duplicated NaN.
`- SPARK-36741: ArrayDistinct should handle duplicated Double.NaN and Float.Nan *** FAILED ***
  Incorrect evaluation: array_distinct([NaN,NaN,null,null,1.0,1.0]), actual: WrappedArray(NaN, NaN, null, 1.0), expected: List(NaN, null, 1.0) (GlutenTestsTrait.scala:263)`

Reply:
Got it, thank you.